reader-NYPL/sw.js

```diff
@@ -0,0 +1,18 @@
+var CACHE_NAME = "webpub-viewer";
```

Comment:
I suggest removing / disabling Service Worker (caching functionality) in Electron